geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -177,6 +177,22 @@ impl GeyserPluginManager {
                 data: None,
             })?;
 
+        // Then see if a plugin with this name already exists. If so, abort
+        if self
+            .plugins
+            .iter()
+            .any(|plugin| plugin.name().eq(new_plugin.name()))
+        {
+            return Err(jsonrpc_core::Error {
+                code: ErrorCode::InvalidRequest,
+                message: format!(
+                    "There already exists a plugin named {} loaded. Did not load requested plugin",
+                    new_plugin.name()
```

Comment:
Can you also include the input `name` in the message? Like "while reloading 'name' "

Reply:
Done